lib/smrt_client.py

```
@@ -0,0 +1,113 @@
+        ).decode("utf-8")
+        payload = {
+            "grant_type": "password",
+            "username": self.username,
+            "password": self.password,
+            "scope": "openid run-design run-qc analysis sample-setup data-management userinfo",
+        }
+        headers = {
+            "Authorization": f"Basic {credentials}",
+            "Content-Type": "application/x-www-form-urlencoded",
+        }
+        r = requests.post(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)